ddtrace/tracer.py

```diff
@@ -531,6 +539,23 @@ def start_span(
         self._hooks.emit(self.__class__.start_span, span)
         return span
 
+    def activate_distributed_header(self, headers, activate=True, propagator=None):
```

Comment:
```suggestion
    def activate_distributed_header(self, *args, **kwargs):
```

(If we decide to make `extract` above also have a generic API).